src/compiler/crystal/tools/doc/method.cr

```diff
@@ -114,15 +114,15 @@ class Crystal::Doc::Method
     String.build { |io| args_to_s io }
   end
 
-  def args_to_s(io)
+  def args_to_s(io : IO) : Nil
     args_to_html(io, links: false)
   end
 
   def args_to_html
     String.build { |io| args_to_html io }
   end
 
-  def args_to_html(io, links = true)
+  def args_to_html(io : IO, links = true) : Nil
```

Comment:
```suggestion
  def args_to_html(io : IO, links : Bool = true) : Nil
```

Reply:
@wontruefree I would definitely use the new commit batch feature for this.

Reply:
@r00ster91 the "Add suggestion to batch" is greyed out for me on this project.  I think it might use a feature like branching that I do not have permission for here.